[2023-12-27 15:42:25,731] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL2.conexion_BD scheduled__2023-12-26T00:00:00+00:00 [queued]>
[2023-12-27 15:42:25,753] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL2.conexion_BD scheduled__2023-12-26T00:00:00+00:00 [queued]>
[2023-12-27 15:42:25,754] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-27 15:42:25,755] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-27 15:42:25,755] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-27 15:42:25,789] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): conexion_BD> on 2023-12-26 00:00:00+00:00
[2023-12-27 15:42:25,799] {standard_task_runner.py:52} INFO - Started process 3777 to run task
[2023-12-27 15:42:25,808] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'OpenWeather_ETL2', 'conexion_BD', 'scheduled__2023-12-26T00:00:00+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/ETL_OpenWeather2.py', '--cfg-path', '/tmp/tmpixu0xxcb', '--error-file', '/tmp/tmp95o6pby4']
[2023-12-27 15:42:25,814] {standard_task_runner.py:80} INFO - Job 76: Subtask conexion_BD
[2023-12-27 15:42:26,001] {task_command.py:371} INFO - Running <TaskInstance: OpenWeather_ETL2.conexion_BD scheduled__2023-12-26T00:00:00+00:00 [running]> on host 44a36bef83ad
[2023-12-27 15:42:26,177] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-12-27 15:42:26,409] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Furtado_Julio
AIRFLOW_CTX_DAG_ID=OpenWeather_ETL2
AIRFLOW_CTX_TASK_ID=conexion_BD
AIRFLOW_CTX_EXECUTION_DATE=2023-12-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-26T00:00:00+00:00
[2023-12-27 15:42:26,413] {logging_mixin.py:115} INFO - Conectandose a la DB en la fecha: 2023-12-26 0
[2023-12-27 15:42:27,687] {logging_mixin.py:115} INFO - <connection object at 0x7fd719cdf590; dsn: 'user=juliocfurtado93_coderhouse password=xxx dbname=data-engineer-database host=data-engineer-cluster.cyhh5bfevlmn.us-east-1.redshift.amazonaws.com port=5439', closed: 0>
[2023-12-27 15:42:27,689] {logging_mixin.py:115} INFO - Conexion a RedShift exitosa!
[2023-12-27 15:42:27,692] {python.py:173} INFO - Done. Returned value was: None
[2023-12-27 15:42:27,794] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=OpenWeather_ETL2, task_id=conexion_BD, execution_date=20231226T000000, start_date=20231227T154225, end_date=20231227T154227
[2023-12-27 15:42:27,874] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-27 15:42:28,018] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
